drop-boxes/register-fastq-dropbox/README.md

```diff
@@ -0,0 +1,33 @@
+#QBiC-register-fastq-dropbox
+
+##Resulting data model in openBIS
+Q_TEST_SAMPLE -> Q_NGS_RAW_DATA (with sample code) -> DataSet (directory with files contained)
```

Comment:
What do you want to indicate here? Do you want to indicate which files to use?
Is it possible to add a diagram to indicate this and textually describe what you want to show here?

Reply:
Also here maybe add example OpenBis codes. I think they must end with `_NGS` ?

Reply:
Ah wait. You are registering the data to the sample. then maybe the code for the sample at the correct level is better

Reply:
Yes, this is just how the entities are connected in openBIS. I don't know if this is necessary for the user in the end, but it was like that for the previous dropbox/etl script.